config.json

```diff
@@ -618,16 +591,22 @@
         "uuid": "95b2f13e-e59f-477c-905a-8cfa652607b3",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 6,
-        "topics": []
+        "difficulty": 6
+      },
+      {
+        "slug": "knapsack",
+        "name": "Knapsack",
+        "uuid": "4aaa8cbf-a48e-475a-a986-bd2cc3468e67",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
       }
     ]
   },
   "foregone": [
     "bank-account",
     "parallel-letter-frequency"
   ],
```

Comment:
Thanks for undoing the remove, but this is still at the root key, instead of within the "exercises" key.

Reply:
Apologies, I misunderstood the comment earlier